cmake/download/dependency_solver.cmake

```diff
@@ -113,6 +112,9 @@ function (RESOLVE_DEPENDENCY NAME_OF_CMAKE_VAR)
     set (FOLDER FALSE)
   endif()
 
+  if (NOT DEFINED SHA256)
+    message(WARNING "SHA is not specified for: " ${NAME_OF_CMAKE_VAR})
```

Comment:
I will replace to FATAL_ERROR after private components update with hash sums

Reply:
Do we have a ticket for that?

Reply:
No need. I have an access to all the necessary private components to change by myself. We need to merge this PR first because extra CMake filed is introduced for RESOLVE_DEPENDENCY function.